src/views/TabView/TabBarBottom.js

```diff
@@ -51,11 +53,16 @@ type Props = {
   tabStyle?: ViewStyleProp,
   showIcon?: boolean,
   isLandscape: boolean,
+  initialLayout?: Layout,
```

Comment:
There is no need for an `initialLayout` prop. If an `initialLayout` prop is provided to `TabNavigator`, the `layout` prop will contain that data.

Reply:
Ah, good to know